.github/workflows/run-tests.yaml

```diff
@@ -41,6 +41,9 @@ jobs:
     steps:
     - uses: actions/checkout@v3
 
+    - name: Inject slug/short variables
+      uses: rlespinasse/github-slug-action@v4
```

Comment:
Need to be careful with the external github action, this could still some env secret like the pypi credential (probably not in this workflow but you get the idea). At the very least we should specify the commit sha and not a major version.

Reply:
I do

How about `rlespinasse/github-slug-action@d1ca8ffbce40ea2371e35bca356db0acacf567d2`?

Reply:
Seems to be safe:

* https://github.com/rlespinasse/github-slug-action/blob/d1ca8ffbce40ea2371e35bca356db0acacf567d2/action.yml
* https://github.com/rlespinasse/slugify-value/blob/v1.4.0/slugify.sh

Reply:
> Sounds good. What does this action do, could we do without (by experience reviewing changes later can be time consuming) ?

https://github.com/pylint-dev/pylint-pytest/actions/runs/6566180670/job/17836304020#step:7:17

tl;dr I'd need to write GH-Actions to do `.replace("/", "_")` 🤦 (because Github Actions are ... un-developed)

_For free_, I get GIT_SHORT_SHA (instead of the `${{ github.sha }}` which is always 40-character)

Reply:
```suggestion
      uses: rlespinasse/github-slug-action@d1ca8ffbce40ea2371e35bca356db0acacf567d2
```

Reply:
You'd need to write that here https://github.com/pylint-dev/pylint-pytest/settings/actions (I don't have access)

Reply:
A replace doesn't seem that bad compared to having to review the next version of this actions, but I'll let you choose what to do.

Reply:
Ha, we would need to allow it for the whole org too. Could we use the replace then ? Are there a lot of replace to add ?

Reply:
No - I just did it. Might be more prudent like that anyway.